homeassistant/components/peco/sensor.py

```diff
@@ -0,0 +1,110 @@
+"""Sensor component for PECO outage counter."""
+from datetime import timedelta
+from types import MappingProxyType
+from typing import Any
+
+import async_timeout
+
+from homeassistant.components.sensor import SensorEntity, SensorStateClass
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+    UpdateFailed,
+)
+
+from .const import _LOGGER, DOMAIN, SCAN_INTERVAL
+from .peco_outage_api import BadJSONError, HttpError, InvalidCountyError, PecoOutageApi
+
+PARALLEL_UPDATES: int = 0
```

Comment:
You don't need to annotate this with `int` - it is inferred anyway. You could however annotate it `typing.Final`.